### PR TITLE
Fix /operations/deliverable-analyzer/start's OpenAPI

### PR DESCRIPTION
**Pull request by akridl**
### Checklist:

* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java [hunk 1]
```diff
@@ -154,7 +153,8 @@ DeliverableAnalyzerOperation updateDeliverableAnalyzer(
                             content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
     @GET
     @Path("/deliverable-analyzer/{id}")
-    @Consumes(MediaType.APPLICATION_JSON_PATCH_JSON) // workaround for PATCH support
+    @Consumes(MediaType.APPLICATION_JSON_PATCH_JSON)
+    // workaround for PATCH support
```
Comment (janinko) [inline rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java:157]: why this change?

Comment (akridl) [inline rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java:157]: Maybe put back the old formatting here? This reformatting makes less sense than the original

Comment (akridl) [reply rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java]: I was just testing your attention, nice job catching that, Honza!

**Review by janinko — APPROVED** [review]
